MediaRecorder.bs

```diff
@@ -241,6 +249,32 @@ interface MediaRecorder : EventTarget {
     the value might be surpassed, not achieved, or only be achieved over a long
     period of time.</li>
 
+    <li>Let |videoKeyFrameIntervalDuration| be the value of |recorder|'s internal
+    [=[[VideoKeyFrameIntervalDuration]]=] slot if defined, and let
+    |videoKeyFrameIntervalCount| be the value of |recorder|'s internal
+    [=[[VideoKeyFrameIntervalCount]]=] slot if defined. The UA SHOULD constrain the
+    configuration of |recorder| so that the video encoders follow the below rules:
+
+    <ol>
```

Comment:
I think unnumbered steps would be better here since they aren't sequential.
```suggestion
    <ul>
```

Reply:
Done.